bouncy/release-bionic-arm64-build.yaml

```diff
@@ -0,0 +1,90 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_label: buildagent_arm64 || bouncy_binarydeb_ubv8
+jenkins_binary_job_priority: 82
```

Comment:
The arm jobs should have the same priority as amd.

Reply:
They use the packet machines so it shouldn't be an issue.
I copied what we currently do for ardent. I can change them to have different priorities but we need to go back to modify previous distros (at least ardent that's still active) accordingly

Reply:
Ok, if they are physically separate the priority doesn't make any difference. :+1: